### PR TITLE
Fixes pipeline failures for awscli-eks-cl2loadtest-with-addons

### PR DESCRIPTION
**Pull request by ashishranjan738**
This commit fixes pipeline failures for awscli-eks-cl2loadtest-with-addons

Signed-off-by: Ashish Ranjan <rnshis@amazon.com>

### CODE REVIEW

File: tests/pipelines/eks/awscli-cl2-load-with-addons.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ spec:
   - name: slack-hook
   - name: slack-message
   - name: amp-workspace-id
+  - name: kubernetes-version
+    default: "1.23"
```
Comment (hakuna-matatah) [inline tests/pipelines/eks/awscli-cl2-load-with-addons.yaml:19]: I thought we have this default at task level IIRC unless it was removed recently ?

Comment (ashishranjan738) [reply tests/pipelines/eks/awscli-cl2-load-with-addons.yaml:19]: yeah its there.

Comment (hakuna-matatah) [reply tests/pipelines/eks/awscli-cl2-load-with-addons.yaml:19]: Then can we remove it from here ? We don't need this redundancy.

Comment (ashishranjan738) [reply tests/pipelines/eks/awscli-cl2-load-with-addons.yaml]: removed

**Review by hakuna-matatah — APPROVED** [review]

**Pull request merged**